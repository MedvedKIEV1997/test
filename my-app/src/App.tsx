import { Routes, Route } from 'react-router-dom';
import { Employees, Search } from './components/index';

function App() {
    return (
        <Routes>
            <Route path="/" element={<Search />}>
                <Route path="/:id" element={<Employees />} />
            </Route>
        </Routes>
    );
}

export default App;
