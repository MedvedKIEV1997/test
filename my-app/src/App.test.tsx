/**
 * @jest-environment-options {"url": "http://localhost:8000/"}
 */

import { renderWithProviders } from './test-utils';

import { BrowserRouter } from 'react-router-dom';
import App from './App';

it('should render app', async () => {
    const { asFragment } = await renderWithProviders(
        <BrowserRouter>
            <App />
        </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
});
