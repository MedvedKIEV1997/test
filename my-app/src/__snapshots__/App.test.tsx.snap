// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render app 1`] = `
<DocumentFragment>
  <div
    class="p-5 container-fluid"
  >
    <div
      class="row"
    >
      <div
        class="col-xl-3 col-lg-4 col-md-5 col-12"
      >
        <div
          class="vstack gap-3 minBreakpoint-xs"
        >
          <input
            class="form-control"
            placeholder="Write name here..."
          />
        </div>
      </div>
      <div
        class="col"
      />
    </div>
  </div>
</DocumentFragment>
`;
